src/spatialdata/_core/spatialdata.py

```diff
@@ -1114,8 +1115,30 @@ def write(
         file_path: str | Path,
         overwrite: bool = False,
         consolidate_metadata: bool = True,
+        format: SpatialDataFormat | list[SpatialDataFormat] | None = None,
     ) -> None:
-        """Write to a Zarr store."""
+        """
+        Write the `SpatialData` object to a Zarr store.
+
+        Parameters
+        ----------
+        file_path
+            The path to the Zarr store to write to.
+        overwrite
+            If `True`, overwrite the Zarr store if it already exists. If `False`, `write()` will fail if the Zarr store
+            already exists.
+        consolidate_metadata
+            If `True`, consolidate metadata (write all the metadata in a single file at the root directory of the
+            store). This optimizes the data to be cloud accessible, and it is even a requirement for certain cloud
+            stores (such with S3).
+        format
+            The format to use for writing the elements of the `SpatialData` object. It is recommended to leave this
+            parameter equal to `None` (default to latest format for all the elements). If not `None`, it must be
+            either a format for an element (formats are defined in `spatialdata._io.format`), or a list of formats.
```

Comment:
shall we add `SpatialDataFormat` to the docstrings and create interlink here?

Reply:
Good point, I have updated the docs. More in a comment below.